packages/web/components/trade-tool/index.tsx

```diff
@@ -11,25 +12,27 @@ import { OrderDirection } from "~/hooks/limit-orders";
 
 export interface TradeToolProps {}
 
-export const TradeTool: FunctionComponent<TradeToolProps> = observer(() => {
-  const [tab, setTab] = useState<SwapToolTab>(SwapToolTab.BUY);
+export const TradeTool: FunctionComponent<TradeToolProps> = () => {
+  const [tab, setTab] = useQueryState("tab", parseAsInteger.withDefault(0));
```

Comment:
nit: Do you think we should swap this to use a string instead of an integer?

i.e. `?tab=swap` instead of `?tab=0`

Reply:
We can adjust the enum as necessary

Reply:
Of course! I will do that asap